### PR TITLE
#8155: Add doc for multigammaln

### PR DESCRIPTION
**Pull request by ruthreshx**
### Ticket
[Link to Github Issue](https://github.com/tenstorrent/tt-metal/issues/8155)

### Problem description
Add doc for multigammaln

### What's changed
Tested with ttnn.bfloat16 & TILE layout (Passed),  ttnn.bfloat_8 & ttnn.ROW_MAJOR_LAYOUT(Crashed hence skipped).

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11236467236)
- [x] [Run Sweeps](https://github.com/tenstorrent/tt-metal/actions/runs/11236425478)

### CODE REVIEW

File: tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py [hunk 1]
```diff
@@ -16,10 +16,10 @@
     "batch_sizes": [(1,)],
     "height": [384, 1024],
     "width": [1024, 4096],
-    "input_dtype": [ttnn.bfloat16],
+    "input_dtype": [ttnn.bfloat16, ttnn.bfloat16],
```
Comment (Aswinmcw) [inline tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py:19]: Bfloat8b?

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 2]
```diff
@@ -1433,7 +1433,14 @@ void py_module(py::module& module) {
...
     detail::bind_unary_composite(module, ttnn::log1p, R"doc(Performs log1p function on :attr:`input_tensor`.)doc", "[supported range -1 to 1]");
     detail::bind_unary_composite(module, ttnn::mish, R"doc(Performs mish function on :attr:`input_tensor`, not supported for grayskull.)doc");
-    detail::bind_unary_composite(module, ttnn::multigammaln, R"doc(Performs multigammaln function on :attr:`input_tensor`.)doc", "[supported range 1.6 to inf]");
+    detail::bind_unary_composite(module, ttnn::multigammaln,
+    R"doc(Supported dtypes, layouts, and ranks:
+
+        +----------------------------+---------------------------------+-------------------+
+        |     Dtypes                 |         Layouts                 |     Ranks         |
+        +----------------------------+---------------------------------+-------------------+
+        |    BFLOAT16, BFLOAT8_B     |          TILE                   |      2, 3, 4      |
```
Comment (Aswinmcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1442]: If row major not supported then add it in skip statement in sweep test

**Review by VirdhatchaniKN — CHANGES_REQUESTED** [review]
Is RM supported?

File: tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py (hunk 1 above)
Comment (VirdhatchaniKN) [inline tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py]:
```
def skip(
    batch_sizes,
    height,
    width,
    input_dtype,
    input_memory_config,
    output_memory_config,
    layout,
) -> Tuple[bool, Optional[str]]:
    if layout == ttnn.ROW_MAJOR_LAYOUT:
        return True, "Skipped as ROW_MAJOR_LAYOUT not supported"
    return False, None

```
If RM is not supported, can you skip the test for the same?

**Review by VirdhatchaniKN — CHANGES_REQUESTED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp (hunk 2 above)
Comment (VirdhatchaniKN) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1442]: See [comment](https://github.com/tenstorrent/tt-metal/pull/13162#discussion_r1777196853)

File: tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py (hunk 1 above)
Comment (ruthreshx) [reply tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py]: Changed @Aswinmcw

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by VirdhatchaniKN — APPROVED** [review]
LGTM

**Review by mouliraj-mcw — APPROVED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 3]
```diff
@@ -1433,7 +1433,14 @@ void py_module(py::module& module) {
     detail::bind_unary_composite(module, ttnn::lgamma, R"doc(Performs lgamma function on :attr:`input_tensor`.)doc", "[supported for value greater than 0]");
     detail::bind_unary_composite(module, ttnn::log1p, R"doc(Performs log1p function on :attr:`input_tensor`.)doc", "[supported range -1 to 1]");
     detail::bind_unary_composite(module, ttnn::mish, R"doc(Performs mish function on :attr:`input_tensor`, not supported for grayskull.)doc");
-    detail::bind_unary_composite(module, ttnn::multigammaln, R"doc(Performs multigammaln function on :attr:`input_tensor`.)doc", "[supported range 1.6 to inf]");
+    detail::bind_unary_composite(module, ttnn::multigammaln,
```
Comment (umadevimcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1436]: `R"doc(Performs multigammaln function on :attr:`input_tensor`.)doc", "[supported range 1.6 to inf]"`

A description is removed. Please add it.

**Review by umadevimcw — CHANGES_REQUESTED** [review]

**Review by umadevimcw — APPROVED** [review]

File: tests/sweep_framework/sweeps/eltwise/unary/multigammaln/multigammaln.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+random.seed(0)
+
+
+# Parameters provided to the test vector generator are defined here.
+# They are defined as dict-type suites that contain the arguments to the run function as keys, and lists of possible inputs as values.
+# Each suite has a key name (in this case "suite_1" and "suite_2") which will associate the test vectors to this specific suite of inputs.
+# Developers can create their own generator functions and pass them to the parameters as inputs.
+parameters = {
+    "multigammaln": {
```
[the remark this answers is not in the document]
Comment (ruthreshx) [reply tests/sweep_framework/sweeps/eltwise/unary/multigammaln/multigammaln.py]: Done

**Review by jdesousa-TT — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Pull request merged**